build.gradle

```diff
@@ -29,7 +29,7 @@ apply plugin: 'project-report'
 apply plugin: 'kotlin'
 
 group 'com.coveros.r3z'
-version '0.0.1-SNAPSHOT'
+version ''
```

Comment:
??

Reply:
All this does is add that text to the end of the name of the binary.  For now, since I'm not using it, I keep it empty, but I may use it in the future.